docs/user/faq.rst

```diff
@@ -77,32 +83,164 @@ the C-dependent libraries from your ``install_requires`` in the RTD environment.
+You can also set your project documentation to install your Python project itself as a dependency.
+
+.. seealso::
+
+   :doc:`/builds`
+     An overview of the build process.
+
+   :doc:`/guides/reproducible-builds`
+     General information about adding dependencies and best-practices for maintaining them.
+
+   :doc:`/build-customization`
+     How to customize your builds, for example if you need to build with different tools from Sphinx or
+     if you need to add additional packages for the Ubuntu-based builder.
+
+   :doc:`Configuration File <config-file/v2>`
+     Reference for the main configuration file, `.readthedocs.yaml`
+
+   :ref:`build.apt_packages <config-file/v2:build.apt_packages>`
+     Reference for adding Debian packages with apt for the Ubuntu-based builders
+
+   Other FAQ entries
+      * :ref:`faq:My documentation requires additional dependencies`
+      * :ref:`faq:I get import errors on libraries that depend on C modules`
+
+
+.. Old reference
+.. _My project requires some additional settings:
+
+Can I have access to additional features or settings?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If this is just a dependency issue,
+see :ref:`faq:My documentation requires additional dependencies`.
+
+Read the Docs offers some settings (feature flags) which can be used for a variety of purposes.
+To enable these settings,
+please send an email to support@readthedocs.org and we will change the settings for the project.
+
+.. seealso::
+
+   :doc:`/feature-flags`
+     Reference of all Feature Flags that can be requested.
+
 
 How do I change behavior when building with Read the Docs?
-----------------------------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When RTD builds your project, it sets the :envvar:`READTHEDOCS` environment
 variable to the string ``'True'``. So within your Sphinx :file:`conf.py` file, you
-can vary the behavior based on this. For example::
+can vary the behavior based on this. For example:
+
+.. code-block:: python
 
     import os
-    on_rtd = os.environ.get('READTHEDOCS') == 'True'
+
+    on_rtd = os.environ.get("READTHEDOCS") == "True"
     if on_rtd:
-        html_theme = 'default'
+        html_theme = "default"
     else:
-        html_theme = 'nature'
+        html_theme = "nature"
 
 The :envvar:`READTHEDOCS` variable is also available in the Sphinx build
-environment, and will be set to ``True`` when building on RTD::
+environment, and will be set to ``True`` when building on RTD:
+
+
+.. code-block:: jinja
 
     {% if READTHEDOCS %}
     Woo
     {% endif %}
 
 
+I want comments in my docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+RTD doesn't have explicit support for this.
+That said, a tool like `Disqus`_ (and the `sphinxcontrib-disqus`_ plugin) can be used for this purpose on RTD.
+
+.. _Disqus: https://disqus.com/
+.. _sphinxcontrib-disqus: https://pypi.python.org/pypi/sphinxcontrib-disqus
```

Comment:
We could update this one to mention https://giscus.app/ that seems that have more users now.

Reply:
I've seen this before but haven't given it much consideration WRT documentation projects. If you can write a specific suggestion, that's great. I'll probably merge this PR now, so just DM me or open a new PR :+1: